### DISCUSSION TITLE
calculate EER using pretrained model-verification

### DISCUSSION THREAD

**Discussion by utkarsh-tyagi** [speechbrain/speechbrain#1658] [Q&A]
Hi Team ,

How to calculate eer using pretrained model (ecapa-tdnn) on my own audio dataset(eg diff dataset like librispeech)?

How to use scripts and Please provide me the steps taken to achieve this and evaluate EER  using your pretrained model.?

What are changes required and which script i have to use?

**Comment by goexle** [comment]
@utkarsh-tyagi you can use this script to calculate the EER for the ecapa-tdnn model model:
[recipes/VoxCeleb/SpeakerRec/speaker_verification_cosine.py](https://github.com/speechbrain/speechbrain/blob/c7039d837345e56d34d159cf95675c8d4b933c20/recipes/VoxCeleb/SpeakerRec/speaker_verification_cosine.py)
